volcanic_island_model_demo.ipynb

```diff
@@ -141,7 +126,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.12"
+   "version": "3.7.7"
```

Comment:
I'm not sure how to review this volcanic_island_model_demo.ipynb file. I think Greg set all of this up and it looks like you've just pulled his most recent changes, but haven't edited anything in this file yourself. Is that correct?  Might want to request Greg's review on this one to make sure it's all good.